test/jdk/javax/swing/border/Test4129681.java

```diff
@@ -21,39 +21,58 @@
  * questions.
  */
 
+import java.awt.event.ItemEvent;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JCheckBox;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+
 /*
  * @test
  * @bug 4129681
  * @summary Tests enabling/disabling of titled border's caption
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test4129681.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual Test4129681
  */
 
-import java.awt.BorderLayout;
-import java.awt.event.ItemEvent;
-import java.awt.event.ItemListener;
-import javax.swing.BorderFactory;
-import javax.swing.JApplet;
-import javax.swing.JCheckBox;
-import javax.swing.JLabel;
+public class Test4129681 {
+    private static JLabel label;
 
-public class Test4129681 extends JApplet implements ItemListener {
-    private JLabel label;
+    public static void main(String[] args) throws Exception {
+        String testInstructions = """
+                Turn on the checkbox to disable the label.
+                The test passes if the title of the border
+                is disabled as well as the label.
+                """;
 
-    @Override
-    public void init() {
-        JCheckBox check = new JCheckBox("disable");
-        check.addItemListener(this);
+        PassFailJFrame.builder()
+                .title("Test Instructions")
+                .instructions(testInstructions)
+                .rows(4)
+                .columns(25)
+                .splitUI(Test4129681::init)
+                .build()
+                .awaitAndCheck();
+    }
 
-        this.label = new JLabel("message");
-        this.label.setBorder(BorderFactory.createTitledBorder("label"));
-        this.label.setEnabled(!check.isSelected());
+    public static JComponent init() {
+        JCheckBox check = new JCheckBox("Enable/Disable");
```

Comment:
```suggestion
        JLable label = new JLabel("message");
        JCheckBox check = new JCheckBox("Enable/Disable");
```
And then both are local variables which you can use in lambda expressions.